modules/apps/adaptive-media/adaptive-media-blogs-web/src/main/java/com/liferay/adaptive/media/blogs/web/internal/optimizer/BlogsAMImageOptimizer.java

```diff
@@ -63,28 +63,32 @@ public void optimize(long companyId) {
 
 		int total = count * amImageConfigurationEntries.size();
 
-		AtomicInteger atomicCounter = new AtomicInteger(0);
+		AtomicInteger successCount = new AtomicInteger(0);
```

Comment:
What do you think about renaming this to `successCounter` or `successAtomicCounter`? Same with errorCount.